### ISSUE TITLE
SwiftUI

### ISSUE THREAD

**Issue by theedov** [Daltron/NotificationBanner#403]
Any plans to support SwiftUI? We're on version 4 already :) 

https://github.com/Daltron/NotificationBanner/issues/327

**Comment by ThePredators** [comment]
I think we can consider this library dead by now, since no support fro SwiftUI :/

By the way i used this since 2023 : [SwiftMessages](https://github.com/SwiftKickMobile/SwiftMessages)

**Comment by airstrike** [comment]
> I think we can consider this library dead by now, since no support fro SwiftUI :/
> 
> By the way i used this since 2023 : [SwiftMessages](https://github.com/SwiftKickMobile/SwiftMessages)

Agreed it's probably dead... I actually used [PopupView](https://github.com/exyte/PopupView) which I found to be entirely awesome.

**Comment by ThePredators** [comment]
> > I think we can consider this library dead by now, since no support fro SwiftUI :/
> > By the way i used this since 2023 : [SwiftMessages](https://github.com/SwiftKickMobile/SwiftMessages)
> 
> Agreed it's probably dead... I actually used [PopupView](https://github.com/exyte/PopupView) which I found to be entirely awesome.

I will try it its indeed very beautiful :)
